.github/workflows/backup/addComments.yml

```diff
@@ -0,0 +1,23 @@
+# Add automated comments to the PR
```

Comment:
Example of what this produces: https://github.com/bcgov/restoration-tracker/pull/91#issuecomment-1105590177

Left it in the backup folder since the other deploy actions are there at the moment.